### PR TITLE
Implement Clone for 'static events

### PR DESCRIPTION
**Pull request by semtexzv**
- [x] Tested on all platforms changed
- [x] Compilation warnings were addressed
- [x] `cargo fmt` has been run on this branch
- [x] `cargo doc` builds successfully
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [x] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
Not applicable:
- [x] Created or updated an example program if it would help users understand this functionality
- [x] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

There is no reason why 'static events can't be Clone. This unblocks most use cases for cloning safe events while not changing current design significantly.

Blocking WASM + OpenGL implementation for Amethyst ( See https://github.com/amethyst/amethyst/pull/2040)

### CODE REVIEW

**Comment by est31** [comment]
Ideally, we should wait for #1456 and then just add a derive impl.

**Comment by semtexzv** [comment]
I agree, current design is not ideal, but if no other approach is found, this is a way to get Clone safely for events.

File: src/event.rs [hunk 1]
```diff
@@ -807,8 +921,10 @@ pub enum VirtualKeyCode {
     Multiply,
     Mute,
     MyComputer,
-    NavigateForward,  // also called "Prior"
-    NavigateBackward, // also called "Next"
+    NavigateForward,
+    // also called "Prior"
```
Comment (filnet) [inline src/event.rs]: comment looks misplaced now

File: src/event.rs [hunk 2]
```diff
@@ -807,8 +921,10 @@ pub enum VirtualKeyCode {
...
+    NavigateBackward,
+    // also called "Next"
```
Comment (filnet) [inline src/event.rs]: same

**Comment by azriel91** [comment]
Heya, is it okay to get this merged (and perhaps published) before a big `Event` refactor?

Since a refactor would be a breaking change anyway, I don't see any harm in allowing downstream projects to depend on a published version with this change, with the prospect that they have to migrate to the new `Event` structure (which they have to do anyway).

**Pull request merged**